receiptui/src/test/java/com/hyperwallet/android/ui/receipt/rule/HyperwalletExternalResourceManager.java

```diff
@@ -1,4 +1,4 @@
-package com.hyperwallet.android.ui.transfermethod.rule;
+package com.hyperwallet.android.ui.receipt.rule;
```

Comment:
com.hyperwallet.android.ui.*receipt*.rule inside of   .../hyperwallet/android/ui/*transfermethod*/rule ?